### DISCUSSION TITLE
Can I do different fragmentation on different nodes ?

### DISCUSSION THREAD

**Discussion by Zelmoghazy** [rqlite/rqlite#1251] [Q&A]
I am sorry If this is a naive question , I am very new to the idea of distributed data bases and I need your guidance , I am trying to build a simple distributed database but apply various fragmentations to a table , and put each fragment in a different node , I dont know how hard this task is but I searched everywhere and there are no beginner friendly guides out there , If anyone can point me to a direction where I can learn more about how to do that I will be very grateful .

**Comment by otoolep** [comment]
rqlite does not support placing different shards on different nodes. Also, this is a not the right place for general discussion of distributed databases. You might have more luck with Reddit.

**Reply by Zelmoghazy** [reply]
Thank you for your input. I apologize if my question was not appropriate for this forum.
